### PR TITLE
Fix prevent the ignored nodes into clustering

### PR DESCRIPTION
**Pull request by xinz**

### CODE REVIEW

File: lib/swarm/tracker/tracker.ex [hunk 1]
```diff
@@ -120,9 +120,13 @@ defmodule Swarm.Tracker do
     # wait for node list to populate
     nodelist = Enum.reject(Node.list(:connected), &ignore_node?/1)
 
-    strategy = Node.self
-           |> Strategy.create()
-           |> Strategy.add_nodes(nodelist)
+    strategy =
+      case ignore_node?(Node.self) do
```
Comment (bitwalker) [inline lib/swarm/tracker/tracker.ex]: Can you explain this part of the change in more detail? I don't think it does what you expect it to.

Comment (xinz) [inline lib/swarm/tracker/tracker.ex]: When we register the workers in the ignored nodes, we should not to put the current ignored node in its strategy(ring), or when we register the workers in the ignored nodes, the started workers would run in the ignored nodes, here are the details to reproduce https://github.com/bitwalker/swarm/issues/31, hope it's clear for you.

Thanks for your reply.

Comment (xinz) [inline lib/swarm/tracker/tracker.ex]: Hi @bitwalker , do we have any updates about it? Please kindly let me know if anything I can help, thanks for your time.

**Comment by xinz** [comment]
After some days reading the source code, I think my commit doesn't observe the design practice of Swarm, it focus on the connected nodes(satisfy the nodes white list if existed) to easily scale clustering, and high availability, my user case would not match the current implement.

Please correct me if any missing, I close and cancel this PR.

**Pull request closed without merging**